server/api.js

```diff
@@ -365,11 +376,19 @@ router.get("/tenders", async (req, res) => {
 
 	const countSql = "SELECT COUNT(*) FROM tender";
 	const dataSql = `
-		SELECT id, title, creation_date, announcement_date, deadline, status 
+		SELECT id, title, creation_date, announcement_date, deadline, description, status, closing_date
 		FROM tender 
 		ORDER BY creation_date DESC 
 		LIMIT $1 OFFSET $2
 	`;
+	const bidSql = `
+    SELECT bid.*
+    FROM bid
+    JOIN tender ON bid.tender_id = tender.id
+    ORDER BY tender.creation_date DESC
+    LIMIT $1 OFFSET $2
```

Comment:
Sorry. I think I incorrectly suggested a join here. This endpoint seems to be doing more than it should, but something like this might work for what you want to do:

```suggestion
    SELECT *
    FROM bid
    WHERE bin.tender_id IN($1)
```

And then pass the ids from the previous query to this one